sdk/python/tests/test_client.py

```diff
@@ -103,6 +104,32 @@ def mock_client(self):
         client._serving_url = SERVING_URL
         return client
 
+    @pytest.fixture
+    def mock_client_with_auth(self):
+        client = Client(
+            core_url=CORE_URL,
+            serving_url=SERVING_URL,
+            core_enable_auth=True,
+            core_auth_token=_FAKE_JWT_TOKEN,
+        )
+        client._core_url = CORE_URL
+        client._serving_url = SERVING_URL
+        return client
+
+    @pytest.fixture
+    def secure_mock_client_with_auth(self):
+        client = Client(
+            core_url=CORE_URL,
+            serving_url=SERVING_URL,
+            core_enable_ssl=True,
+            serving_enable_ssl=True,
+            core_enable_auth=True,
```

Comment:
I think you should be enabling security separately here for serving.

Reply:
Please see my comments above.